examples/cloudsuite/kit/cloudsuite.py

```diff
@@ -0,0 +1,494 @@
+            )
+
+            if "Starting MariaDB database server mariadbd" in ret:
+                mariadb_initialized = True
+
+    def single_run(  # pylint: disable=arguments-differ
+        self,
+        benchmark_duration_seconds: int,
+        nb_threads: int = 2,
+        **kwargs,
+    ) -> str:
+
+        ip_web_server = self.web_server_platform.comm.get_ipaddress
+
+        self._clean_run()
+        self._build_run(nb_threads)
+
+        ping_values = self.platform.comm.shell(
+            command=f"ping -c 5 {ip_web_server}",
+            print_input=False,
+            print_output=False,
+        )
+
+        environment = self._preload_env(
+            **kwargs,
+        )
+
+        has_container = self.platform.comm.pipe_shell(
+            command="docker container ls -a | grep faban_client | wc -l", ignore_ret_codes=[1]
+        )
+
+        if "1" in has_container:
+            run_command = [
+                "docker",
+                "start",
+                "-i",
+                "faban_client",
+            ]
+        else:
+            faban_output_dir = self._data_dir / "faban_output"
+            seeds_file = self._data_dir / "_tmp_benchkit_seeds.txt"
+
+            run_command = [
+                "docker",
+                "run",
+                "--net=host",
+                "--name=faban_client",
+                "-v",
+                f"{faban_output_dir}:/web20_benchmark/output",
+                "--env-file",
+                f"{seeds_file}",
+                "faban_built",
+                f"{ip_web_server}",
+                f"{nb_threads}",
+                f"--ramp-up={nb_threads + 20}",
+                f"--steady={benchmark_duration_seconds}",
+                "--oper=run",
+            ]
+
+        wrapped_run_command, wrapped_environment = self._wrap_command(
+            run_command=run_command,
+            environment=environment,
+            **kwargs,
+        )
+
+        output = self.run_bench_command(
+            run_command=run_command,
+            wrapped_run_command=wrapped_run_command,
+            current_dir=self._bench_src_path,
+            environment=environment,
+            wrapped_environment=wrapped_environment,
+            print_output=False,
+        )
+
+        return f"{ping_values}\n{output}"
+
+    def get_details(  # pylint: disable=arguments-differ
+        self,
+        record_data_dir: PathType,
+        nb_threads: int,
+    ) -> None:
+        faban_output_dir = self._data_dir / "faban_output"
+
+        output_dir = faban_output_dir / f"TH_{nb_threads}-TM_1000-TY_THINKTIME-DS_fixed"
+        filename = output_dir / "1/detail.xan"
+
+        file_content = self.platform.comm.shell(
+            f"cat {filename}",
+            print_input=False,
+            print_output=False,
+        )
+
+        self._write_to_record_data_dir(file_content, "detail.xan", record_data_dir)
+
+        self.platform.comm.shell(
+            f"sudo rm -rf {output_dir}",
```

Comment:
```suggestion
            f"sudo mv {output_dir} /tmp/benchkit/.trash",
```

Quick suggestion to avoid catastrophes.